Grammatically updated README.md (#695)
revanced/revanced-manager: README.md

> **Warning**: This repository currently has no active maintainer. For that reason, development is stale. Active development continues for [revanced-cli](https://github.com/revanced/revanced-cli). If you are interested in maintaining this repository, please let us know at manager@revanced.app.

## 🔽 Download

> To download the Alpha version of the Manager, go [here](https://github.com/revanced/revanced-manager/releases/latest) and install the provided APK file.

Reviewer: ```suggestion
The early version of ReVanced Manager can be obtained [here](https://github.com/revanced/revanced-manager/releases/latest).
```

Author: Noted your suggestion. I'm going to submit a major update to the `README.md` soon
